src/components/VideoPlayerContexts/PlaybackContext.tsx

```diff
@@ -69,12 +59,9 @@ function PlaybackContextProvider({children}) {
     );
 
     const checkVideoPlaying = useCallback(
-        (statusCallback) => {
-            if (!(currentVideoPlayerRef && currentVideoPlayerRef.current && currentVideoPlayerRef.current.getStatusAsync)) {
-                return;
-            }
-            currentVideoPlayerRef.current.getStatusAsync().then((status) => {
-                statusCallback(status.isPlaying);
+        (statusCallback: (isPlaying: boolean) => void) => {
```

Comment:
Let's create a separate type for `(isPlaying: boolean) => void`, this way it will be more readable